### PR TITLE
Create package.json

### PR DESCRIPTION
**Pull request by ilyazub**
I'll check tomorrow that everything works with such `package.json` and mark PR as _Ready for review_.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "keywords": [
+    "jquery",
+    "bootstrap",
+    "dual-list-box"
+  ],
+  "author": "alex3305",
+  "contributors": [
+    "patrickfust",
+    "hheimbuerger"
```
Comment (hheimbuerger) [inline package.json]: Add @flo-f (ahead of myself) please.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "license": "ISC",
```
Comment (hheimbuerger) [inline package.json]: The source of our fork appears to be MIT, so let's stick to that.

File: package.json (hunk 1 above)
Comment (ilyazub) [reply package.json]: Done.

**Comment by ilyazub** [comment]
Just checked. Page that uses `dual-list-box` works fine.

**Review by hheimbuerger — CHANGES_REQUESTED** [review]
Please fix the license as noted on another (non-review) comment.

File: package.json (hunk 2 above)
Comment (ilyazub) [reply package.json]: Done.

**Review by hheimbuerger — APPROVED** [review]

**Pull request merged**